pkg/mount/mount.go

```diff
@@ -543,6 +548,33 @@ func (m *Mounter) removeSoftlinkAndTarget(link string) error {
 	return nil
 }
 
+// isPathSetImmutable returns true on error in getting path info or if path
+// is immutable .
+func (m *Mounter) isPathSetImmutable(mountpath string) bool {
+	if _, err := os.Stat(mountpath); err != nil {
+		dlog.Errorf("Failed to stat mount path:%v", err)
+		return true
+	}
+	op, err := exec.Command(lsattrBin, "-d", mountpath).CombinedOutput()
+	if err != nil {
+		// Cannot get path status, return true so that immutable bit is not reverted
+		dlog.Errorf("error listing attrs for %v err:%v", mountpath, string(op))
+		return true
+	}
+	attrs := strings.Split(string(op), " ")
```

Comment:
Can you add a comment on what the output should look like, so that we know what we are trying to parse.

Reply:
yes. good to have the sample output.